### PR TITLE
Hacktoberfest :  Updated & Improved milvus_adopters.md 

### PR DESCRIPTION
**Pull request by iamartyaa**
Sorted the Adopters added later, starting from iYUNDONG, in alphabetical order.
Closes: #609

### CODE REVIEW

**Comment by LocoRichard** [comment]
@iamartyaa Thank you for your contribution! We really appreciate your help. Since the Hacktoberfest starts on Octo. 1st, I will merge your commits till then so that you can get the official Hacktoberfest perks and stuff. I will also assign the corresponding issue to you to avoid others to make the same contribution.

**Comment by iamartyaa** [comment]
Okay! Thank you @LocoRichard .

**Review by claireyuw — CHANGES_REQUESTED** [review]
DXY.cn and Deepset.ai should switch places

File: site/en/about/milvus_adopters.md [hunk 1]
```diff
@@ -8,31 +8,31 @@ Milvus is trusted by over 1,000 organizations worldwide and has applications in
...
 | ------------------------------------------------------------ | ---------------------- | ------------------------------------------------------------ |
+| [ambeRoad](https://amberoad.de/?lang=en) |  Software| |
 | [Beike](https://investors.ke.com/about-us/default.aspx)                                                | Real estate           | [Making With Milvus: AI-Infused Proptech for Personalized Real Estate Search](https://zilliz.com/blog/Making-With-Milvus-AI-Infused-Proptech-for-Personalized-Real-Estate-Search)                                              |
 | [DXY.cn](https://en.wikipedia.org/wiki/DXY.cn)                                                  | Medical           |                                             |
 |[Deepset.ai](https://deepset.ai/)                                                    | Software         | [Semantic Search with Milvus, Knowledge Graph QA, Web Crawlers and more!](https://medium.com/deepset-ai/semantic-search-with-milvus-knowledge-graph-qa-web-crawlers-and-more-837451eae9fa)                                             |
```
Comment (claireyuw) [inline site/en/about/milvus_adopters.md:14]: Please put Deepset.ai in front of DXY.cn

**Review by claireyuw — CHANGES_REQUESTED** [review]

File: site/en/about/milvus_adopters.md (hunk 1 above)
Comment (iamartyaa) [reply site/en/about/milvus_adopters.md:14]: Okay. I am on it !

**Comment by iamartyaa** [comment]
I have made the necessary changes and created a new pull request. Please review it @claireyuw .

**Pull request closed without merging**